cloud_webserver_v2/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+services:
+  app:
```

Comment:
Can you rename app to be more descriptive, we will eventually have more entries in here. Maybe `cloud_webserver_v2`